frontend/public/components/monitoring/receiver-forms/slack-receiver-form.tsx

```diff
@@ -72,6 +85,144 @@ export const Form: React.FC<FormProps> = ({ globals, formValues, dispatchFormCha
+                    })
+                  }
+                  checked={formValues.slackIconType === 'url'}
+                  inline
+                />
+                <RadioInput
+                  title="Emoji"
+                  name="slackIconType"
+                  value="emoji"
+                  data-test-id="slack-icon-type-emoji"
+                  onChange={(e) =>
+                    dispatchFormChange({
+                      type: 'setFormValues',
+                      payload: { slackIconType: e.target.value },
+                    })
+                  }
+                  checked={formValues.slackIconType === 'emoji'}
+                  inline
+                />
+              </label>
+              {formValues.slackIconType === 'url' && (
+                <>
+                  <input
+                    className="pf-c-form-control"
+                    type="text"
+                    aria-describedby="slack-icon_url-help"
+                    data-test-id="slack-icon_url"
+                    value={formValues.slack_icon_url}
+                    onChange={(e) =>
+                      dispatchFormChange({
+                        type: 'setFormValues',
+                        payload: { slack_icon_url: e.target.value },
+                      })
+                    }
+                  />
+                  <div className="help-block" id="slack-icon_url-help">
+                    The url of the icon
+                  </div>
+                </>
+              )}
+              {formValues.slackIconType === 'emoji' && (
+                <>
+                  <input
+                    className="pf-c-form-control"
+                    type="text"
+                    aria-describedby="slack-icon_emoji-help"
+                    name="slackIconEmoji"
+                    data-test-id="slack-icon_emoji"
+                    value={formValues.slack_icon_emoji}
+                    onChange={(e) =>
+                      dispatchFormChange({
+                        type: 'setFormValues',
+                        payload: { slack_icon_emoji: e.target.value },
+                      })
+                    }
+                  />
+                  <div className="help-block" id="slack-icon_emoji-help">
+                    An{' '}
+                    <ExternalLink
+                      href="https://www.webfx.com/tools/emoji-cheat-sheet/"
+                      text="emoji code"
+                    />{' '}
+                    string to use in place of the default icon
+                  </div>
+                </>
+              )}
+            </div>
+            <div className="form-group">
+              <label className="control-label" htmlFor="slackUsername">
+                Username
+              </label>
+              <input
+                className="pf-c-form-control"
+                type="text"
+                aria-describedby="slack-username-help"
+                id="slackUsername"
```

Comment:
IDs should be kebab case

```suggestion
                id="slack-username"
```